viewer/templates/viewer/help.html

```diff
@@ -28,7 +34,7 @@ <h2>Title</h2>
 
 <div class="block">
   <h2>URL</h2>
-  <p>Find all instances of a string in the URL of a published cf.gov page.</p>
+  <p>Find all instances of a string in the URL of a cf.gov page. </p>
   <p class="u-bold_text">Examples</p>
```

Comment:
Is it intentional that the introductory text does wrap while the examples don't?

<img width="978" alt="image" src="https://user-images.githubusercontent.com/654645/184199340-5a9e1765-d2a3-413e-96aa-0f31424cd35d.png">

Reply:
I specifically changed the `max-width` of the bullet points because we didn't want the links wrapping, but this is the standard `max-width` wrapping in the paragraphs